src/update-feeds-loops/submit-transactions.ts

```diff
@@ -167,10 +168,21 @@ export const estimateMulticallGasLimit = async (
     // Adding a extra 10% because multicall consumes less gas than tryMulticall
     return (goEstimateGas.data * BigInt(Math.round(1.1 * 100))) / 100n;
   }
-  logger.warn(`Unable to estimate gas for multicall using provider.`, { errorMessage: goEstimateGas.error.message });
+  const errorMessage = goEstimateGas.error.message;
+  // It is possible that the gas estimation failed because of a contract revert due to timestamp check, because the feed
+  // was updated by other provider in the meantime. Try to detect this expected case and log INFO instead.
+  if (errorMessage.includes('Does not update timestamp')) {
+    logger.info(`Gas estimation failed because of a contract revert.`, { errorMessage });
+  } else {
+    logger.warn(`Unable to estimate gas for multicall using provider.`, { errorMessage });
+  }
 
   if (!fallbackGasLimit) {
-    throw new Error('No fallback gas limit provided.');
```

Comment:
Looks like we don't need to utilize promise-utils while calling this function `estimateMulticallGasLimit` above.

Reply:
Yeah, thx.